code/modules/photography/camera/camera.dm

```diff
@@ -121,6 +121,9 @@
 			return FALSE
 		else if(!(get_turf(target) in get_hear(world.view, user)))
 			return FALSE
+	else if(istype(loc, /mob/living/carbon))
```

Comment:
```suggestion
	else if(isliving(loc))
```

Reply:
Now uses isliving checks in place of all istype carbon checks.